metricbeat/module/kubernetes/state_persistentvolumeclaim/_meta/fields.yml

```diff
@@ -0,0 +1,28 @@
+- name: persistentvolumeclaim
+  type: group
+  description: >
+    kubernetes persistent volume clame metrics from kube-state-metrics
+  release: experimental
+  fields:
+    - name: name
+      type: keyword
+      description: PVC name.
+    - name: volume_name
+      type: keyword
+      description: Binded volume name.
+    - name: request_storage.bytes
+      type: long
+      description: Requested capacity.
+    - name: phase
+      type: keyword
+      description: PVC phase.
+    - name: access_mode
+      type: keyword
+      description: Access mode.
+    - name: storage_class
+      type: keyword
+      description: Storage class for the PVC.
+    - name: labels.*
```

Comment:
do you know if this labels are exactly Kubernetes labels? If that's the case we could map them under the common `kubernetes.labels`

Reply:
It is.

I moved it to the module location. 
Will need to do so with PV and Services (on a different PR)